deploy/kubernetes/driver/kubernetes/manifests/controller-server.yaml

```diff
@@ -155,6 +160,29 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
+        - name: storage-secret-sidecar
```

Comment:
Opensource lib should not have sidecar related changes, can you remove all sidecar related reference